src/TimeSlotGroup.js

```diff
@@ -46,7 +46,8 @@ export default class TimeSlotGroup extends Component {
       // The exception to the above observation is for the label column on the left
       // of the calendar, which is why we will re-render on value updates for
       // *only* those columns in particular.
-      this.props.showLabels && date.neq(this.props.value, nextProps.value)
+      this.props.showLabels && date.neq(this.props.value, nextProps.value) ||
+      (!this.props.showLabels && this.props.availabilities !== nextProps.availabilities)
```

Comment:
I think this is definitely a safer change (from a performance perspective at least) than the previous change that compared `value`s, though it does mean we need to carefully memoize `props.availabilities` on both the hippo side and the react-big-calendar side or else this could completely break the optimizations here because if the reference to `props.availabilities` keeps changing even when the actual availabilities haven't, `this.props.availabilities !== nextProps.availabilities` will evaluate to true when it shouldn't.

An example of where I suspect referential equality isn't stable, the default `[]` fallback here:
https://github.com/elationemr/react-big-calendar/blob/ad21877a80985bcd07c36c0cb348033a08589909/src/MultiTimeGrid.js#L245
In situations where there is no availability map, this falls back to setting `availabilities` equal to an empty array, but that empty array is a new reference on every render because it's declared inside of the function.

However, I played around with this in Ellie and didn't actually see the behavior I was expecting to see (which is that when no availabilities exist, individual time slots would re-render for no reason). I'm a bit far removed from calendar code at this point so I'll just note that the performance here is something to keep an eye on.

For what it's worth, a more robust way to do this would be to compare the dates/times contained within the availabilities array rather than a strict referential equality check. This is similar to what's being done on line 49 right above this for `props.value`: two Date objects might represent the same date but have different references in memory, which would cause `this.props.value !== nextProps.value` to incorrectly evaluate to true. That's why the code does `date.neq()` instead of `!==`, because it compares the actual values contained in the Dates rather than their references, and the check will only evaluate to true if the dates are meaningfully different.